### PR TITLE
Mulitple modifications asked by PO

### PR DESCRIPTION
**Pull request by lespacedunmatin**

### CODE REVIEW

**Review by GoOz — APPROVED** [review]
Looks good to me 👍

File: src/styles/app.styl [hunk 1]
```diff
@@ -470,6 +470,7 @@ footer
     display flex
     justify-content center
     align-items stretch
+    margin-top 27px
```
Comment (gregorylegarec) [inline src/styles/app.styl]: @GoOz : is it ok to have a dimension in pixels here ? Where does `27` comes from ?

Comment (GoOz) [reply src/styles/app.styl]: oh right. Missed this one. The thing is there's no Cozy-UI on this repo, so we can't use functions like em()… Sooooo until then let's use em by doing do the math ourselves

**Comment by gregorylegarec** [comment]
@lespacedunmatin On Chrome and Firefox, MAIF service thumbnail layout is broken.
capture d ecran 2017-04-19 a 14 04 15

**Comment by gregorylegarec** [comment]
Also, I am not sure about Orange and EDF icon size, but I cannot check the mockup right now. I'll update my comment as soon as possible.

File: src/styles/app.styl (hunk 1 above)
Comment (lespacedunmatin) [reply src/styles/app.styl]: Ok, I do the modification. 27 is the value that makes the design match de request from Zoe.

**Comment by gregorylegarec** [comment]
@lespacedunmatin I confirm that the logos are too large compared to the mockups. We should limit their height to have an aspect like MAIF one. EDF logo should be larger than Orange's one, so again I think it is just a matter of height.

**Review by gregorylegarec — CHANGES_REQUESTED** [review]
See my comments in main conversation.

**Comment by lespacedunmatin** [comment]
I noticed that the logos where too big but I didn’t  want to make hazardous modifications on pieces I know litttle about. I will make the modifications.

**Comment by gregorylegarec** [comment]
Thanks @lespacedunmatin, I think it's better but not yet coherent with the mockups and with the MAIF service aspect. I think we should use the same mechanism than the MAIF logo : https://github.com/cozy/cozy-onboarding-v3/blob/master/src/styles/app.styl#L517

I suggest:
```stylus
.service-logo--orange
    box-sizing border-box
    width 4.25rem
    height 4.25rem

.service-logo--edf
    box-sizing border-box
    height 4.25rem
```
And in `view_step_maif.jade`:
```jade
div(class="services")
      .service
        if edfLogo
          figure.service-logo.service-logo--edf
            svg: use(xlink:href=edfLogo)
        .service-name= t('step edf service')
        .service-actions
          button= t('step edf connect')
      .service
        if orangeLogo
          figure.service-logo.service-logo--orange
            svg: use(xlink:href=orangeLogo)
        .service-name= t('step orange service')
        .service-actions
          button= t('step orange connect')
```

**Comment by lespacedunmatin** [comment]
Oh, I was wondering how to add that class. But it would be a tailored solution that would need a special work for every logo, that’s not very generic…

**Comment by gregorylegarec** [comment]
@lespacedunmatin I agree, that's the main problem with logos, they all have specific dimensions. Do you think about an alternative ?

**Comment by lespacedunmatin** [comment]
Ok, I heavily messed the log with my rebase…

**Comment by gregorylegarec** [comment]
@lespacedunmatin Thanks that better now. Last think EDF icon is still too thin. Its `max-width` value should be `7rem`.

And we have a last request from @kelukelu, the `.service-name` element should have a `margin-bottom` equal to `1rem`. May you do this last change please ?

And don't worry about the history tree, I think that after the merge with master, it will be simpler to squash all the commits before merging.

EDIT : better for now all `.service-logo` may have a `max-width` of `7rem` and keeps an `height` of `4.25rem`

**Comment by lespacedunmatin** [comment]
Oh crap, the new rebase is even worse…

**Comment by lespacedunmatin** [comment]
@gregorylegarec 8431e87 brings the modifications you submitted.

**Review by gregorylegarec — APPROVED** [review]

**Comment by gregorylegarec** [comment]
Thanks @lespacedunmatin, I'll try to squash all the commit to get a more readable tree. 👍

**Pull request merged**